### DISCUSSION TITLE
Welcome to freeDictionaryAPI Discussions!

### DISCUSSION THREAD

**Discussion by meetDeveloper** [meetDeveloper/freeDictionaryAPI#99] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AmjadHD** [comment]
Hi, thank you for your work on this, it really is helpful 👍🙂
I'm developing a simple sublime text plugin:
![image](https://user-images.githubusercontent.com/33235928/138550504-db36b2a8-8ce6-40b8-a1bf-4cde1c086d31.png)
and I have a question about the `phonetics` key: why is it an array ? can you provide an example where it holds more than one object ?
